python/doc/examples/numerical_methods/general_methods/plot_study_save_load.py

```diff
@@ -71,6 +71,7 @@
 #
 # - its name: it is useful to give names to the objects we want to save. If no name has been given by the user, we can use the name by default. The name of each object saved can be checked in the file.XML created or by printing the study in the python interface (with the command print).
 # - its id number : this id number is unique for each object. It is useful to distinguish objects of same type which names are identical, equal to the default name given ('Unnamed'). This id number may be checked by printing the study loaded in the python interface (with the command print): be careful, this print operation must be performed after having loaded the study (the id number may be different from the one indicated in the file.XML associated to the study).
+# - for HDF5 storage : the id is used to name the HDF5 dataset. This way, the various floating point/integer collections are stored and accessed through the name of their respective datasets.
```

Comment:
```suggestion
# - for HDF5 storage: the id is used to name the HDF5 dataset. This way, the various floating point/integer collections are stored and accessed through the name of their respective datasets.
```
Does this mean that the id serves as both id and name, or that the HDF5 dataset has no name?

Reply:
So does the HDF5 dataset have no name?

Reply:
The dataset gets named after the id, as shown in the code snippet below. id role in the xml remains unchanged
```  
<object class="Point" id="9" size="1" StudyVisible="false" StudyLabel="">
    <string>/Study_0.h5:/9</string>
  </object>```

Reply:
I updated this part to better reflect this

Reply:
Thanks for your explanation.